lib/entities/map/ValetudoMap.js

```diff
@@ -31,7 +31,10 @@ class ValetudoMap extends SerializableEntity { //TODO: Current, Historic, Etc.
         this.layers = [];
         this.entities = [];
 
-        this.metaData.version = 1; //Will probably be incremented some day
+        this.metaData = {
```

Comment:
Are you sure that this is required?

Reply:
I will check it again, but I'm pretty sure

Reply:
Oh sorry I didn't realize that the SerializableEntity handles the metaData